src/test/integration/fixtures/build-with-preset/expected/es5-bundled/shared_bundle_1.html

```diff
@@ -1 +1 @@
-<template>I'm baz.html!</template>
\ No newline at end of file
+<html><head><template>I'm baz.html!</template></head><body></body></html>
```

Comment:
Not sure why these phantom dom5 tags have re-appeared, since I thought we were quite careful to remove them both in bundler and in the build HTML transform. From offline conversation, I think we're going to leave these for the upcoming pre release and file an issue to investigate.

Reply:
https://github.com/Polymer/polymer-cli/issues/991